pages/edge/cli.mdx

```diff
@@ -1,217 +1,206 @@
 import { Callout } from "nextra-theme-docs";
 
-# `wasmer app` and `wasmer deploy` CLI
+# The `app` CLI subcommand
+Wasmer provides you with CLI commands to deploy and manage your applications on the Edge, 
+namely the `wasmer app` subcommand.
 
-Wasmer provides you with commands in their CLI to deploy and manage your applications on the edge.
-
-## Usage
-
-Wasmer has two main commands for deploy:
-
-- `wasmer app`
-- `wasmer deploy`
-
-<Callout type="info" emoji="ℹ️">
-  **Note:** You must be logged in to Wasmer to use the `app` and `deploy`
-  commands.
+<Callout type="info">
+  **Note:** To deploy apps on Wasmer Edge, users need an account. When
+  deploying an app, users are prompted to login if they did not already. 
 </Callout>
 
-### `wasmer app`
-
-It allows the user to:
-
-- Create an app
-- Get information about an app
-- List all of their apps
-
-#### `wasmer app create`
-
-This command is used to create an app that will be deployed on the edge.
-
-This further provides you with options:
-
-1. Use an existing package, like a `static-web-server` for static website.
-2. Create your own custom app from scratch.
-
-You can also specify an app type with the `--type` flag. The available types are:
-
-- Static website
-- HTTP server
-- Browser shell **(coming soon)**
+## Create
 
-#### `wasmer app get`
-
-This command is used to get the information about an app.
-
-**Arguments**
-
-- `ident`: The identifier of the app you want to get information about. This is an optional argument. If you do not provide an app identifier, it will automatically get that from your `app.yaml` file in your current directory. This is basically the `name` field in your `app.yaml` file.
-
-**Options**
-
-- `--format`: The format to use for the output. This defaults to `yaml`. Other options are `json` and `text`.
-- `--token`: The token to use for authentication. You can also use the `WASMER_TOKEN` environment variable. This defaults to the token in your `~/.wasmer/wasmer.toml` file.
-- `--verbose`: Show verbose output.
-- `--quiet`: Do not print progress messages.
-- `--color`: Whether to use colored output. Defaults to `auto`. Possible values are `auto`, `always`, and `never`.
+Before deploying an app, one must be created: to this end, the wasmer CLI
+provides the `wasmer app create` command.
```

Comment:
We'd like to make the process very simple for users, that means that knowing one command is better than two.

```suggestion
Before deploying an app, one must be created: to this end, the first time
you run `wasmer deploy` a new app will be created (or via `wasmer app create`).
```